for_darwin_to_detect_compiler_flag.patch

```diff
@@ -1,23 +0,0 @@
---- a/configure.ac	2019-04-19 16:27:52.000000000 +0900
-+++ b/configure.ac	2019-04-19 16:26:06.000000000 +0900
-@@ -39,6 +39,20 @@ AC_PROG_LN_S
- dnl Compiling with per-target flags requires AM_PROG_CC_C_O.
- AC_PROG_CC
- AM_PROG_CC_C_O
-+
-+# Check if a compiler supports "-Wno-error=address-of-packed-member"
```

Comment:
This patch doesn't work on macOS as expected. For example,  https://travis-ci.org/frugalos/liberasurecode/jobs/564276849.